src/AWSAPI.jl

```diff
@@ -376,77 +365,59 @@ function service_request_function(service)
     else
         operation = ""
     end
-    push!(args,
-            "args         = args")
-
-"""
-function $name(aws::AWSConfig, $(verb)$(resource)$(operation)args=[])
-
-    AWSCore.service_$protocol(
-        aws;
-        $(join(args, ",\n        ")))
-end
-
-$name($(verb)$(resource)$(operation)args=[]) =
-    $name(default_aws_config(), $(verb)$(resource)$(operation)args)
-
-$name(a...; b...) = $name(a..., b)
-"""
```

Comment:
What happened to this?

Reply:
This code was commented out, I just removed it since it's not being used.